LayoutTests/imported/w3c/web-platform-tests/navigation-api/navigate-event/signal-abort-detach-in-onnavigate-expected.txt

```diff
@@ -1,5 +0,0 @@
-
-Harness Error (TIMEOUT), message = null
```

Comment:
I think deleting this file was a mistake.

Reply:
Well spotted, fixed.